arm/Microsoft.RecoveryServices/vaults/readme.md

```diff
@@ -59,38 +60,9 @@ This module deploys a recovery service vault.
 | `userAssignedIdentities` | object | `{object}` |  | The ID(s) to assign to the resource. |
 
 
-### Parameter Usage: `backupStorageConfig`
+### Parameter Usage: `<ParameterPlaceholder>`
```

Comment:
A lot of parameter usage snippets got lost in the readme update. Please double check and add them back.

Reply:
Done